uimaj-core/src/main/java/org/apache/uima/resource/impl/PearSpecifier_impl.java

```diff
@@ -38,38 +38,36 @@
   private String mPearPath;
 
   private Parameter[] mParameters;
+  private NameValuePair[] mPearParameters;
 
   /**
    * Creates a new <code>PearSpecifier_impl</code>.
    */
   public PearSpecifier_impl() {
   }
 
-  /**
-   * @return Returns the Parameters.
-   */
+  @Deprecated
   public Parameter[] getParameters() {
     return this.mParameters;
   }
+  
+  public NameValuePair[] getPearParameters() {
+    return this.mPearParameters;
+  }
 
-  /**
-   * @param parameters
-   *          The Parameters to set.
-   */
-  public void setParameters(Parameter[] parameters) {
+  @Deprecated
+  public void setParameters(Parameter... parameters) {
     this.mParameters = parameters;
   }
+  
+  public void setPearParameters(NameValuePair... pearParameters) {
+    this.mPearParameters = pearParameters;
+  }
 
-  /* (non-Javadoc)
-   * @see org.apache.uima.resource.PearSpecifier#getPearPath()
-   */
   public String getPearPath() {
     return this.mPearPath;
   }
 
-  /* (non-Javadoc)
-   * @see org.apache.uima.resource.PearSpecifier#setPearPath(java.lang.String)
-   */
```

Comment:
not sure: the javadoc was removed, but no @Deprecated follows?  Should this have a deprecated?  should the javadoc also be left in, but say @deprecated with a ref to what to do instead?

Reply:
See comment below.